js/bybit.js

```diff
@@ -4696,19 +5003,124 @@ module.exports = class bybit extends Exchange {
         }
         let type = undefined;
         [ type, params ] = this.handleMarketTypeAndParams ('fetchClosedOrders', market, params);
+        const request = {};
```

Comment:
@sc0Vu same question here, `fetchSpotClosedOrders` uses v3 only, what if the user has a unified account v5?